src/meshapi/models/__init__.py

```diff
@@ -1,6 +1,6 @@
-from .building import *
```

Comment:
nit: I think there is a way to do this based on [file path globs](https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-per-file-ignores). Something like:

```
per-file-ignores =
    **/__init__.py:F401,F403
```

in `.flake8`

Reply:
Might also want to add `src/meshapi_hooks/models.py`?